myriad-scheduler/src/main/java/com/ebay/myriad/scheduler/event/handlers/ResourceOffersEventHandler.java

```diff
@@ -136,7 +139,7 @@ public void onEvent(ResourceOffersEvent event, long sequence,
     }
   }
 
```

Comment:
This is getting clunky, as to my previous comment one profiles Maybe we define like this:
Profiles:
  -nodemanager:
       -cpu: 1
       -mem: 1024
       -ports:
          -yarn.nodemanager.webapp: -1
          -yarn.nodemanager.rpc: -1
          ....
  jobhistoryserver:
    -cpu: 1
    -mem: 1024
    -ports:
       -web: 8000
       -rpc: 8031
     ...

Then if its -1 for a port we get random, if it positives we get exactly that one.  This is the 30 sec version it'll need expanded.